src/modals/ValidatorMetrics/index.tsx

```diff
@@ -99,45 +78,8 @@ export const ValidatorMetrics = () => {
               </StatWrapper>
             )
           )}
-        </StatsWrapper>
+        </ValidatorMetricsStatsWrapper>
       </PaddingWrapper>
-      <div
-        className="body"
-        style={{ position: 'relative', marginTop: '0.5rem' }}
-      >
-        <SubscanButton />
-        <GraphWrapper
-          style={{
-            margin: '0 1.5rem 0 0.5rem',
-            height: 350,
-            border: 'none',
-            boxShadow: 'none',
-          }}
-          flex
-        >
-          <h4>
-            Recent Era Points <OpenHelpIcon helpKey="Era Points" />
```

Comment:
Decision from @DamianStraszak was to remove the era points graph because it's confusing to users. We show them data they will not be able to correctly interpret.